### DISCUSSION TITLE
MS SQL backend

### DISCUSSION THREAD

**Discussion by saucyknave** [jan-vandenberg/cruddiy#140] [General]
I just discovered Cruddiy and am testing it out.  I have a Microsoft SQL server on site that I'd like to use with Cruddiy, but it throws MySQL-looking errors when I start it up and try to connect to my MS SQL server.  Can someone please confirm whether or not Cruddiy works with a Microsoft SQL Server backend?  Thanks.

**Comment by jan-vandenberg** [comment]
Hi!

Cruddiy does not work with MS SQL.
